App.tsx

```diff
@@ -1,20 +1,78 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { useState } from "react";
+import { StyleSheet, View, FlatList, ScrollView } from "react-native";
+import TodoItem from "./components/TodoItem";
+import TodoInput from "./components/TodoInput";
+import { Image } from "react-native";
 
 export default function App() {
+  const [todosList, setTodosList] = useState([]);
+
+  function addTodoText(enteredTodoText) {
+    if (enteredTodoText != "") {
+      setTodosList([
+        ...todosList,
+        { text: enteredTodoText, id: Math.random().toString() },
+      ]);
+    }
+  }
+
   return (
     <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-      <StatusBar style="auto" />
+      <View>
+        <Image source={require("./assets/favicon.png")} style={styles.images} />
```

Comment:
This Image component is not expo-image please fix.

Reply:
mashaAllah yes perfect! in the future exercise this week, please explore caching rather then doing doing inline `require`.